Cargo.toml

```diff
@@ -45,3 +45,6 @@ bitcoin_hashes = "0.9"
 [target.wasm32-unknown-unknown.dev-dependencies]
 wasm-bindgen-test = "0.3"
 rand = { version = "0.6", features = ["wasm-bindgen"] }
+
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
I did some testing locally where I deleted this and it doesn't seem to have an impact?

I can build and run the tests on wasm without this being specified.

After doing some digging, I found [this](https://stackoverflow.com/questions/56227766/why-must-a-wasm-library-in-rust-set-the-crate-type-to-cdylib). I _think_ we only need to set this for the crate that actually uses `wasm_bindgen` to export functions from a WASM module.

Unless someone wants to directly use the functions provided in this library from JS (i.e. without creating a new crate and declaring a dependency on this one), setting this here is just noise and doesn't actually do anything useful I believe.

Reply:
Did not pass CI without specifying crate-type